crates/loader/src/bindle/mod.rs

```diff
@@ -106,6 +107,15 @@ async fn prepare(
     })
 }
 
+fn validate_raw_app_manifest(raw: &RawAppManifest) -> Result<()> {
+    let _ = raw
+        .components
+        .iter()
+        .map(|c| validate_allowed_http_hosts(&c.wasm.allowed_http_hosts))
+        .collect::<Result<Vec<_>>>()?;
+    Ok(())
```

Comment:
Minor shortening; feel free to ignore:

```suggestion
    raw
        .components
        .iter()
        .try_for_each(|c| validate_allowed_http_hosts(&c.wasm.allowed_http_hosts))?
```

Reply:
Ooh, TIL. Thanks!